crates/typst/src/ide/complete.rs

```diff
@@ -360,6 +360,15 @@ fn field_access_completions(ctx: &mut CompletionContext, value: &Value) {
         })
     }
 
+    for &field in fields_on(value.type_name()) {
+        ctx.completions.push(Completion {
+            kind: CompletionKind::Constant,
+            label: field.into(),
+            apply: None,
+            detail: None,
+        })
+    }
+
```

Comment:
(This is fine, right?)

Reply:
You could use `value_completion` instead of doing a manual completion. Then, the completion will not just show the name of the field, but also its concrete value.